response_operations_ui/views/reporting_units.py

```diff
@@ -325,6 +326,7 @@ def search_reporting_units():
     )
 
 
+# Not sure why is this endpoint exposed : Amit Sinha
```

Comment:
Are you not sure why it's exposed because nothing uses it or something else? Do you want to make a quick investigation card so we can look into it?